docs/config.rst

```diff
@@ -1,11 +1,623 @@
+        steps:
+            1:
+                task: command
+                options:
+                    command: echo 'Hello there!' 
+            2:
+                task: util_sleep
+
+This is a flow comprised of two tasks: ``command`` greets the user by echoing a string, and ``util_sleep`` then tells CumulusCI to sleep for five seconds. You can reference how the flows are defined in the universal ``cumulusci.yml`` file `here <https://github.com/SFDO-Tooling/CumulusCI/blob/d038f606d97f50a71ba1d2d6e9462a249b28864e/cumulusci/cumulusci.yml#L565>`_.
+
+
+Add a Flow Step
+^^^^^^^^^^^^^^^
+
+If you want to add a step to a flow, you first need to know what the existing steps are. The easiest way to see this is with ``cci flow info <name>``.
+
+    Example: This is the ``dev_org`` flow.
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All **non-negative numbers and decimals** are valid as step numbers in a flow.
+This makes it easy to add steps before, in-between, or after existing flow steps.
+If we wanted to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1. Example values would include 0, 0.3, and 0.89334. All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
+
+If you want to add a step **between** steps 2 and 3, then a step number of 2.5 can be used.
+
+If you want to add a step **after** all steps in the flow, then any step number greater than 4 can be used.
+
+You can add an additional log line at the end of the ``dev_org`` flow with the following under the ``flows`` section of your project's ``cumulusci.yml`` file:
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            5:
+                task: log
+                    options:
+                        line: dev_org flow has completed
```

Comment:
Let's make this a separate example. As we discussed, it doesn't really blend well with the prior couple of lines.
```suggestion
For example, you can add an additional log line outpu at the end of the ``dev_org`` flow by adding the following under the ``flows`` section of your project's ``cumulusci.yml`` file:

.. code-block:: yaml

    dev_org:
        steps:
            5:
                task: log
                    options:
                        line: dev_org flow has completed
```